testing/kuttl/e2e-other/exporter-upgrade/02--cluster.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: exporter
+spec:
+  postgresVersion: 14
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.5-1
```

Comment:
I think we should remove the defined image and let `RELATED_IMAGES` take over. Assuming we can remove things from the spec with KUTTL :thinking: :pick:

Reply:
For now, and with the understanding that this test is odd because the image is hard-coded, I would rather leave it hard-coded just so we know what the error we expect will be. (Or to be totally honest: I feel like this test isn't going to be run as is very often, but may serve as a blueprint for a better version and/or as documentation for what we might expect.)